src/main/java/frc/robot/RobotContainer.java

```diff
@@ -109,8 +110,8 @@ private void configureButtonBindings() {
     m_driverController.button(OIConstants.RIGHT_BUMPER).whileTrue(new Intake(IntakeConstants.SCORE_MID_SPEED));
     m_driverController.button(OIConstants.LEFT_BUMPER).whileTrue(new Intake(IntakeConstants.SCORE_LOW_SPEED));
     m_driverController.leftTrigger().whileTrue(new Intake(IntakeConstants.INTAKE_SPEED));
-    m_driverController.button(OIConstants.A_BUTTON).onTrue(new PivotDown(IntakeConstants.PIVOT_DOWN_SPEED));
-    m_driverController.button(OIConstants.X_BUTTON).onTrue(new PivotUp(IntakeConstants.PIVOT_UP_SPEED));
+    m_driverController.button(OIConstants.A_BUTTON).onTrue(new PivotPIDCommand(18));
```

Comment:
Is 18 a setpoint? If not what does it mean. Make it a variable name of what it means

Reply:
Changed the former constant PIVOT_MAX and MIN to equal our setpoints and used that in the commands